types/react-scroll-to-bottom/react-scroll-to-bottom-tests.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from "react";
+import ScrollToBottom, {
+	FunctionContext, StateContext, useAnimating, useAnimatingToEnd, useAtBottom, useAtEnd,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/react-scroll-to-bottom/react-scroll-to-bottom-tests.tsx;
  Line: 3;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.